fvtest/tril/tril/ast.hpp

```diff
@@ -59,9 +78,15 @@ struct ASTValue {
     enum ASTType {Integer, FloatingPoint, String} ;
 
     // aliases for the underlying C++ types used to store the different AST types
+#if __cplusplus <= 199711L // Check for C++11 compiler support.
+    typedef uint64_t Integer_t;
```

Comment:
I think I'd just keep the typedefs, rather than ifdefs. It doesn't bring extra value preserving the the `using`